app/helpers/diaper_drive_helper.rb

```diff
@@ -0,0 +1,7 @@
+module DiaperDriveHelper
+  def is_virtual(diaper_drive)
```

Comment:
```suggestion
  def is_virtual(diaper_drive: )
```
This is a bit of a minor detail, but I suggest using keyword arguments. It isn't always clear to me what I should put in as a parameter using this approach.

Thoughts?

Reply:
I agree, good suggestion